pandas/core/arrays/timedelta.py

```diff
@@ -83,6 +101,52 @@ def __new__(cls, values, freq=None):
 
         return result
 
+    @classmethod
+    def _generate(cls, start, end, periods, freq, closed=None, **kwargs):
```

Comment:
we call this ``_generate_range`` above, maybe in future can share *some* code in base class